pkg/apis/devops/v1alpha3/pipeline_types.go

```diff
@@ -77,6 +77,15 @@ const (
 	MultiBranchPipelineType = "multi-branch-pipeline"
 )
 
+const (
+	SourceType_SVN       = "svn"
```

Comment:
In convention, we use camel case. I think SourceTypeSVN is good enough.

Reply:
Yes.